content/en/integrations/guide/aws-manual-setup.md

```diff
@@ -109,17 +113,14 @@ For more details, see the [How to use an external ID when granting access to you
 
 ### Datadog
 
-3. Open the [AWS integration tile][2].
+3. In the [AWS integration tile][2], click **Add AWS Account**, and then select **Manually**.
 4. Select the **Access Keys (GovCloud or China Only)** tab.
-5. Enter your `AWS Access Key` and `AWS Secret Key`. Only access and secret keys for GovCloud and China are accepted.
-6. Choose the services to collect metrics from on the left side of the dialog.
-7. Optionally, add tags to all hosts and metrics.
-8. Optionally, monitor a subset of EC2 instances by entering the AWS tags in the textbox `to hosts with tag`. **Note:** This also applies to an instance's attached EBS volumes.
-9. Optionally, monitor a subset of Lambdas by entering the AWS tags in the textbox `to Lambdas with tag`.
-10. Click **Install Integration**.
+5. Enter your `Account ID`, `AWS Access Key` and `AWS Secret Key`. Only access and secret keys for GovCloud and China are accepted.
+6. Click **Save**.
+7. Wait up to 10 minutes for data to start being collected, and then view the out-of-the-box <a href="https://app.datadoghq.com/screen/integration/7/aws-overview" target="_blank">AWS Overview Dashboard</a> to see metrics sent by your AWS services and infrastructure.
```

Comment:
use regular markdown link?

Reply:
As above, this is a workaround to have the link open in a new tab.